dev/releases/make_github_release.py

```diff
@@ -22,6 +23,9 @@
     utils.error("Could not get GAP version")
 
 utils.notice(f"Detected GAP version {GAPVERSION}")
+if re.fullmatch( r"[1-9]+\.[0-9]+\.[0-9]+", GAPVERSION) == None:
+    utils.error("This does not look a release version")
+
```

Comment:
Does this take care of the TODO on line 51 of the GitHub workflow?

Reply:
Note quite (that TODO should be clarified, sorry): while this at least catches cases where I ended up uploading release to tags like v4.12dev, it still doesn't catch the scenario where the CI build is for tag v5.0.0 but then for some reason we end up picking version v6.7.8 here (granted, this is unlikely in practice, it just happened a lot in my tests when I was too eager pushing more test tags 😂 ). So it may not be worth the effort of addressing it.. *shrug* Just feels like a good idea to have lots of extra sanity checks to prevent weird stuff down the road that none of us can foresee right now...

Reply:
I'm all for sanity checks!

Reply:
I'm gonna dig out the tag code I wrote during the GAP Days.